src/TLuaInterpreter.cpp

```diff
@@ -309,6 +309,11 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
             pE.mArgumentList.append( QString(lua_tostring( L, i )) );
             pE.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
         }
+        else if( lua_isboolean( L, i ) )
```

Comment:
Right, and booleans can be coerced to strings as well (i.e., tostring(true) returns "true" and tostring(false) returns "false"), but lua_isstring doesn't count them as auto-convertible. As I understand it, the reason for that is that the coercion is not symmetric: If you cast false to a string, you get "false", but if you cast "false" to a boolean, you get true.